apply-cluster.sh

```diff
@@ -16,6 +16,10 @@ USAGE="PROJECT=<projectid> CLUSTER=<cluster> $0"
 PROJECT=${PROJECT:?Please provide project id: $USAGE}
 CLUSTER=${CLUSTER:?Please provide cluster name: $USAGE}
 
+# Version numbers for Helm and Helm charts.
+K8S_HELM_VERSION="v3.3.0"
+K8S_INGRESS_NGINX_VERSION="4.2.1"
```

Comment:
We define these exact variables and values in [apply-global-prometheus.sh](https://github.com/m-lab/prometheus-support/blob/main/apply-global-prometheus.sh#L27). Might we consider creating a separate file for variables of this type that both apply scripts import to avoid duplication? It seems there could be other things in the variables file like `OAUTH_PROXY_CLIENT.*` variables?

Reply:
I moved some of the common varables to `config.sh` and now both scripts source it. Thoughts?